docs/openapi/components/schemas/credentials/GS1DataCredential.yml

```diff
@@ -0,0 +1,126 @@
+$linkedData:
+  term: GS1DataCredential
+  '@id': https://w3id.org/traceability#GS1DataCredential
+title: GS1 Data Credential
+tags:
+  - eCommerce
+description: >-
+  A data credential is data about something identified with a key credential. While anyone can make any declaration about an object, for that data to be trusted, it has to be authorised in some fashion. This can vary depending on the type of data and the business process in which it is used. For example, a brand owner can be trusted to declare the brand name, description, size and unit of measure, and much more about their own product. The brand owner may not be trusted, however, to declare certain product certifications (e.g., Halal and conformance to other religious practices) or to provide high-quality data for critical supply chain processes (e.g., dimensions and weights for planogram purposes). In such cases, the brand owner would have to authorise other parties that are trusted to provide that data.
+  On the other hand, an anonymous consumer can provide a review of the product without any authorisations provided by the brand owner but can increase trust in the review by referencing a proof of purchase credential for the product.
+  Ultimately, the rules governing which declarations may be made, by whom, and with what combination of data authorisation and data certification credentials depend on the data being provided.
+  The key authorisation and data certification credential URIs do not need to be resolvable, and the fact that this credential is presented does not require that the other credentials be presented as well. It is up to the holder to decide whether to present the other credentials alongside this one, and up to the verifier to decide whether or not to accept them (or accept their absence).
+type: object
+properties:
+  '@context':
+    type: array
+    readOnly: true
+    const:
+      - https://www.w3.org/2018/credentials/v1
+      - https://w3id.org/traceability/v1
+      - https://w3id.org/vc/status-list/2021/v1
+    default:
+      - https://www.w3.org/2018/credentials/v1
+      - https://w3id.org/traceability/v1
+      - https://w3id.org/vc/status-list/2021/v1
+    items:
+      type: string
+      enum:
+        - https://www.w3.org/2018/credentials/v1
+        - https://w3id.org/traceability/v1
+        - https://w3id.org/vc/status-list/2021/v1
+  type:
+    type: array
+    readOnly: true
+    const:
+      - VerifiableCredential
+      - GS1DataCredential
+    default:
+      - VerifiableCredential
+      - GS1DataCredential
+    items:
+      type: string
+      enum:
+        - VerifiableCredential
+        - GS1DataCredential
+  id:
+    description: A DID for this credential. The ID is not resolvable because the credential is sensitive information and only the user company can decide when it is presented.
```

Comment:
does this have to be a DID,? can it not be a normal ID?

Reply:
As per the spec, yes. I'll run it by GS1...

Reply:
PS: Can we agree that DIDs are also "normal"  :)